src/sass/main.css

```diff
@@ -8,14 +8,21 @@ body {
   background-color: #E5E5E5;
 }
 
+body::-webkit-scrollbar {
```

Comment:
Z tego co kiedyś czytałem to raczej nie powinno się stosować:
https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar

Ale chyba to kompilator sam dorzucił :)